dist/windows/qt.conf

```diff
@@ -2,4 +2,3 @@
 Translations = translations
 
 [Platforms]
-WindowsArguments = dpiawareness=1
```

Comment:
I would comment it out since this option is hard to find on the net.

Reply:
And users might want to tweak it (for me at least).

Reply:
In what case one would want not to use a per monitor dpi awareness?

Reply:
>In what case one would want not to use a per monitor dpi awareness?

To turn it off.

Reply:
I have the same opinion, better to comment it out, qt ui looks and behaves pretty good with default awareness setting on win, I have tested this lot.

Reply:
> In my case it's not scaling good enough.

That's sound like another issue, probably worse than having blurred fonts if that's the way you prefer.
If that's true dpiawareness should remain '1'.
What exactly wrong with DPI scaling in your case, does it have an open issue ?

Reply:
>That's sound like another issue, probably worse than having blurred fonts if that's the way you prefer.
>What exactly wrong with DPI scaling in your case, does it have an open issue ?

It's not that bad and its more like a personal preference (the scaling somehow breaks the aesthetic that I rather have blurry UI).

Reply:
> I would comment it out since this option is hard to find on the net.  

Done (PR rebased).
I was looking for the ill DPI scaling, but didn't find anything.
It might be a good idea to open an issue depicting the bad scaling, ideally the program should always run with per monitor DPI awareness.